lib/get-config.js

```diff
@@ -296,13 +300,13 @@ function getRuleFromString(rule) {
   const indexOfSeparator = rule.indexOf(':') + 1;
 
   // we have to split based on the index of the first instance of the separator because the separator could exist in the second half of the rule
-  const name = rule.substring(0, indexOfSeparator - 1);
-  let ruleConfig = rule.substring(indexOfSeparator);
+  const name = rule.slice(0, Math.max(0, indexOfSeparator - 1));
```

Comment:
For this specific case, I’m not sure if the code looks better using slice instead of substring ^^

Reply:
That's true. I reverted this change for now.

Note that we can remove `Math.max` if we know the index will always be positive.